en/django_installation/instructions.md

```diff
@@ -165,6 +165,8 @@ In the console, run `pip install --upgrade pip`.
 
 Then run `pip install django~=1.9.0` (note that we use a tilde followed by an equal sign: `~=`) to install Django.
 
+**NOTE:** When you see command-line examples like the following you only need to type what is written after the `$` character (or `>` on Windows). The other lines represents the output of the command you entered.
```

Comment:
It's a great idea but you should probably put this the first time they encounter the command line. It's in [python instruction file](https://github.com/DjangoGirls/tutorial/blob/master/en/python_installation/instructions.md).